pyproject.toml

```diff
@@ -1,6 +1,37 @@
+[build-system]
+requires = ["setuptools>=61.2", "setuptools_scm[toml]>=3.4.3"]
+build-backend = "setuptools.build_meta"
+
 [project]
 name = "ntia_conformance_checker"
-version = "0.0.1"
-dependencies = [
-    "spdx-tools>=0.7.0rc0",
+version = "0.2.0"
+authors = [{name = "John Speed Meyers", email = "TBD@gmail.com"}]
```

Comment:
```suggestion
authors = [{name = "John Speed Meyers", email = "johnmeyersster@gmail.com"}]
```

What is `authors` vs. `maintainers`? I don't particularly consider myself an author of this project.